Source/WebInspectorUI/UserInterface/Controllers/TimelineManager.js

```diff
@@ -86,6 +86,7 @@ WI.TimelineManager = class TimelineManager extends WI.Object
 
         if (target.hasDomain("Timeline")) {
             // COMPATIBILITY (iOS 13): Timeline.enable did not exist yet.
+            // COMPATIBILITY (iOS X.Y, macOS X.Y): `Timeline.enable` did not exist yet for Worker targets.
```

Comment:
Won't this fail at `target.hasDomain("Timeline")` in this case?

I believe we'd want to put iOS 18.4 since that's the last versioned protocol file without "worker" targetType for Timeline domain.

Reply:
oh hmm yeah good catch.  ill change it

we normally only update the compatibility comments once the next protocol version is cut as that's the cleanest way to know for sure when it was first shipped